---
bool door_is_open
